guideforms.py

```diff
@@ -120,6 +120,12 @@
                  'in the feature list. Anyone with the link will be able to '
                  'view the feature on the detail page.')),
 
+    'spec_repo_url': forms.URLField(
+      required=False, label='Spec repo',
+      widget=forms.URLInput(attrs={'placeholder': 'https://'}),
+      help_text=('Link to spec repository.  This is where the spec will be '
+                 'kept under version control even if it does not exist yet.')),
```

Comment:
I'm a little confused by this line. How can something be kept under version control if it doesn't exist? How can I have a URL for it?